### ISSUE TITLE
Using Typemill as a static website generator

### ISSUE THREAD

**Issue by Tekl** [typemill/typemill#349]
Well, Typemill is quite fast and the caching works well. But it would be cool to have a way to generate/update a complete static website with Typemill.

**Comment by trendschau** [comment]
Yes, that was an idea for typemill version 2, but it is a long way to go....

**Comment by trendschau** [comment]
I close this, it is out of scope for Typemill
